package.json

```diff
@@ -63,9 +63,8 @@
     }
   },
   "volta": {
-    "node": "14.21.3",
+    "node": "16.20.1",
```

Comment:
unrelated, but we should probably drop node 14 -- separate PR tho

Reply:
approved 👍